packages/web/hooks/ui-config/use-asset-info-config.ts

```diff
@@ -212,6 +216,12 @@ export class ObservableAssetInfoConfig {
 
   protected _disposers: (() => void)[] = [];
 
+  @observable
+  denom: string;
+
+  @observable
+  coinMinimalDenom?: string;
```

Comment:
Does this need to be observable state, or can it just be readonly properties (defined statically as `readonly` in constructor params)

Reply:
Good point, it can be an observable because we are also using a memo, so if they change we'll have a new instance of `ObservableAssetInfoConfig`.
I'll refactor it

Reply:
Done